cms/djangoapps/contentstore/rest_api/v1/views/tests/test_course_index.py

```diff
@@ -61,8 +61,8 @@ def test_course_index_response(self):
                 "blocks": [],
                 "advance_settings_url": f"/settings/advanced/{self.course.id}"
             },
-            "discussions_incontext_feedback_url": "",
-            "discussions_incontext_learnmore_url": "",
+            "discussions_incontext_feedback_url": "https://discuss.openedx.org/t/new-and-improved-discussions-forum/9183",  # lint-amnesty, pylint: disable=line-too-long
+            "discussions_incontext_learnmore_url": "https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/manage_discussions/discussions.html",  # lint-amnesty, pylint: disable=line-too-long
```

Comment:
Did you intend to use the settings you defined in common.py here, instead of hard-coding these URLs?

Reply:
Definitely!